src/swaps/astroport/AstroportSwapProvider.ts

```diff
@@ -188,12 +189,8 @@ class AstroportSwapProvider extends SwapProvider {
   // ======== UTILS ========
 
   _getRPC() {
-    const { chainID, nodeUrl } = TerraNetworks.terra_mainnet;
-
-    return new LCDClient({
-      chainID,
-      URL: nodeUrl,
-    });
+    const { chainId, rpcUrl } = TerraNetworks.terra_mainnet;
+    return new LCDClient({ chainID: String(chainId), URL: String(rpcUrl) });
```

Comment:
I think we can remove `String(...)` from `chainId` and `rpcUrl` because they are strings by default: https://github.com/liquality/chainify/blob/7fc75bbd9a603c7fd38181a07a4e2e8ce5a67906/packages/terra/lib/networks.ts#L3

Reply:
<img width="655" alt="image" src="https://user-images.githubusercontent.com/7030722/170506224-966dd972-a888-4c7b-a81d-528763f6ae91.png">
<img width="644" alt="image" src="https://user-images.githubusercontent.com/7030722/170506288-690c79d7-171b-4560-adad-567878b6a87b.png">